.circleci/config.yml

```diff
@@ -41,14 +41,17 @@ commands:
       - run:
           name: Install dependencies
           command: |
+            sudo wget -qO - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo tee /etc/apt/pg_gpg_key
+            sudo apt-key add /etc/apt/pg_gpg_key
+            echo "deb http://apt.postgresql.org/pub/repos/apt/ focal-pgdg main" | sudo tee -a /etc/apt/sources.list
+            sudo apt-get update && sudo apt-get install -y postgresql-11
```

Comment:
This line isn't in the snovault circle config?

Reply:
https://github.com/ENCODE-DCC/snovault/pull/344/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R34 etc